src/test/kotlin/com/amazon/ionelement/IonElementLoaderTests.kt

```diff
@@ -45,6 +50,8 @@ class IonElementLoaderTests {
         // Converting from IonValue to IonElement should result in an IonElement that is equivalent to the
         // parsed IonElement
         assertEquals(parsedIonValue.toIonElement(), parsedIonElement)
+
+        assertEquals(tc.expectedElement, parsedIonElement)
```

Comment:
🗺️ I was surprised to discover that there was previously no assertion that the parsed element was equal to the expected element. 😮